File: src/main/java/com/leijendary/spring/boot/template/api/v1/data/SampleTranslationResponse.java
package com.leijendary.spring.boot.template.api.v1.data;

import com.leijendary.spring.boot.template.data.LocaleData;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class SampleTranslationResponse extends LocaleData {

    private String name;
    private String description;
}

File: src/main/kotlin/com/leijendary/spring/template/api/v1/model/SampleTranslationResponse.kt
package com.leijendary.spring.template.api.v1.model

import com.leijendary.spring.template.core.model.LocaleModel

data class SampleTranslationResponse(
    val name: String,
    val description: String,
    override val language: String,
    override val ordinal: Int
) : LocaleModel(language, ordinal)
